CMakeLists.txt

```diff
@@ -91,6 +92,7 @@ include_directories("${FMT_SRC}/include")
 include_directories("${TREELITE_SRC}/include")
 include_directories("${TREELITE_SRC}/runtime/native/include")
 include_directories("${PROJECT_SOURCE_DIR}/include")
+include_directories("${JSON_SRC}/single_include")
```

Comment:
@apivovarov  we are adding a single_include here. Do you mean instead of cloning the entire repo we just clone this one file?